code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/style/AepButtonStyle.kt

```diff
@@ -67,7 +69,8 @@ class AepButtonStyle(
                 shape = overridingStyle.shape ?: defaultStyle.shape,
                 border = overridingStyle.border ?: defaultStyle.border,
                 colors = overridingStyle.colors ?: defaultStyle.colors,
-                contentPadding = overridingStyle.contentPadding ?: defaultStyle.contentPadding
+                contentPadding = overridingStyle.contentPadding ?: defaultStyle.contentPadding,
+                buttonTextStyle = AepTextStyle.merge(defaultStyle.buttonTextStyle ?: AepTextStyle(), overridingStyle.buttonTextStyle)
```

Comment:
should this merge just be
```kotlin
buttonTextStyle = overridingStyle.buttonTextStyle ?: defaultStyle.buttonTextStyle
```
to match the merge behavior of the other parameters?

Reply:
Ideally we should be doing the same to all styles composed of multiple fields like TextStyle but it didn't seem to work with system provided method. I'll try to explore if there's a consistent way to solve this